### PR TITLE
Binary example documentation imrpvements

### PR DESCRIPTION
**Pull request by Sammers21**
Related to #361.

### CODE REVIEW

File: examples/binary/README.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 [![](https://github.com/artipie/artipie/workflows/Proof::binary/badge.svg)](./examples/binary)
 
-Try this `repo.yaml` file:
+This directory contains a basic example of how to can use artipie as a storage for binary files.
```
Comment (olenagerasimova) [inline examples/binary/README.md:5]: @Sammers21 please, correct second part of the sentence, subject is missing there

File: examples/binary/README.md [hunk 2]
```diff
@@ -12,6 +15,21 @@ repo:
     path: /var/artipie/data
 ```
 
-You can send HTTP PUT/GET requests
-to `http://localhost:8080/repo/<filename>` to upload/download a binary file,
-e.g. `http://localhost:8080/repo/libsqlite3.so`.
+After creating the configuration file below, artipie is ready to server as a storage for binary files.
```
Comment (olenagerasimova) [inline examples/binary/README.md:18]: @Sammers21 second part of the sentence seems odd, correct, please

File: examples/binary/README.md [hunk 3]
```diff
@@ -12,6 +15,21 @@ repo:
...
+
+In order to download a file, send a GET HTTP request:
+
+```bash
+curl -X GET http://localhost:8080/repo/text.txt
+```
+
+#### Advanced option
+
+The binary type of repositories does not have any other opinions available.
```
Comment (olenagerasimova) [inline examples/binary/README.md:35]: @Sammers21 I'd suggest to change "binary type of repositories" -> "binary repository type"

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@Sammers21 thanks, added several comments. Also, I think we should choose the way we write "artipie" in our docs: either always with the capital letter or not and fix it all over the dosc...

File: examples/binary/README.md (hunk 1 above)
Comment (olenagerasimova) [inline examples/binary/README.md:5]: @Sammers21 
part "how to can use artipie as a storage for binary files" is still not correct, I'd suggest "how artipie can be used as ... "

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@Sammers21 one more comment, please, take a look

**Review by olenagerasimova — APPROVED** [review]
@Sammers21 thanks
@olegmoz looks good to me

**Review by olegmoz — APPROVED** [review]

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#567` is not assigned, can't get performer

**Pull request merged**